dash_core_components/_imports_.py

```diff
@@ -1,47 +1,47 @@
-from .Checklist import Checklist
 from .ConfirmDialog import ConfirmDialog
+from .Input import Input
+from .Upload import Upload
+from .LogoutButton import LogoutButton
 from .ConfirmDialogProvider import ConfirmDialogProvider
+from .Link import Link
+from .Location import Location
+from .Tab import Tab
+from .SyntaxHighlighter import SyntaxHighlighter
 from .DatePickerRange import DatePickerRange
+from .Checklist import Checklist
 from .DatePickerSingle import DatePickerSingle
-from .Dropdown import Dropdown
 from .Graph import Graph
-from .Input import Input
+from .RadioItems import RadioItems
+from .Store import Store
 from .Interval import Interval
-from .Link import Link
-from .Location import Location
-from .LogoutButton import LogoutButton
 from .Markdown import Markdown
-from .RadioItems import RadioItems
+from .Dropdown import Dropdown
 from .RangeSlider import RangeSlider
-from .Slider import Slider
-from .Store import Store
-from .SyntaxHighlighter import SyntaxHighlighter
-from .Tab import Tab
-from .Tabs import Tabs
 from .Textarea import Textarea
-from .Upload import Upload
+from .Tabs import Tabs
+from .Slider import Slider
 
 __all__ = [
-    "Checklist",
     "ConfirmDialog",
+    "Input",
+    "Upload",
+    "LogoutButton",
     "ConfirmDialogProvider",
+    "Link",
+    "Location",
+    "Tab",
+    "SyntaxHighlighter",
     "DatePickerRange",
+    "Checklist",
     "DatePickerSingle",
-    "Dropdown",
     "Graph",
-    "Input",
+    "RadioItems",
+    "Store",
     "Interval",
-    "Link",
-    "Location",
-    "LogoutButton",
     "Markdown",
-    "RadioItems",
+    "Dropdown",
     "RangeSlider",
-    "Slider",
-    "Store",
-    "SyntaxHighlighter",
-    "Tab",
-    "Tabs",
     "Textarea",
-    "Upload"
+    "Tabs",
+    "Slider"
```

Comment:
These were sorted before, now they're not. What changed?

Reply:
The 'transpiler' was rewritten in Python, and the sort order changed relative to the R version. I will correct this; sorry for the unnecessary diffs this generated...

Reply:
OK, I've looked into this. The reshuffled order isn't present when cloning the repo, but occurs after running `dash-generate-components ./src/components/ dash-core-components`, with or without the `--r-prefix`. 

It seems to be outside of the R component generation methods, but I might be mistaken.

@T4rk1n @rmarren1

Reply:
if that's the case, we should make a separate PR that commits that batch of changes, so we can review/approve the Python and R changes separately :)

Reply:
@alexcjohnson By commenting about this change, you revealed two distantly related issues that should be addressed. Thanks -- talking with @rmarren1, this is probably related to swapping out `_get_metadata` for `json.loads`, and inadvertently not using an `OrderedDict` anymore.

I'm going to verify this, will then open PR for Dash if true, and link back to here.